internal/commands/util.go

```diff
@@ -110,6 +110,20 @@ func ServerStateColour(state string) text.Colors {
 	}
 }
 
+// FormatServerState implements Format function for server states
```

Comment:
Is this good place for this format function or should it be in `ui` package, similarly than `FormatPossiblyUnknownString`.

Reply:
Created new format package for these. We could later also move/add LB and DB state formatters there.